canonical-web-and-design/maas-ui
# Controller override failed testing

## Done

- Make the override failed testing form reusable.
- Add the override failed testing form to the controller list header.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to the machine list and tick a machine that has failed testing.
- Use the take action menu to open the override failed testing and submit. It should work.
- Repeat the same for the controller list (you might not be able to do this easily depending on your MAAS setup).

## Fixes

Fixes: https://github.com/canonical-web-and-design/app-tribe/issues/618.
